scripts/sync_mephisto_task.py

```diff
@@ -32,18 +34,42 @@ def run_replace():
     with open(MEPHISTO_TASK_PACKAGE) as mephisto_task_package:
         version = json.load(mephisto_task_package)["version"]
 
-    print(f"Updating files to use mephisto-task version {version}")
+    is_check_mode = len(sys.argv) > 1 and sys.argv[1] == "check"
+    are_all_synced = True
+
+    print(f"Detected mephisto-task version '{version}' at '{MEPHISTO_TASK_PACKAGE}'")
+    if is_check_mode:
+        print(
+            f"Checking all dependent files are using mephisto-task version '{version}'...\n"
+        )
+    else:
+        print(f"Syncing all dependent files to mephisto-task version '{version}'...\n")
```

Comment:
Added a bit more verbose logging, also different messages depending on what mode the script is running in